ampd/Cargo.toml

```diff
@@ -73,6 +79,10 @@ url = "2.3.1"
 valuable = { version = "0.1.0", features = ["derive"] }
 valuable-serde = { version = "0.1.0", features = ["std"] }
 voting-verifier = { workspace = true }
+aleo-gateway = { workspace = true }
+snarkvm-cosmwasm = { workspace = true }
+# aleo-parser = { path = "../../axelar-aleo/crates/aleo-parser" }
+aleo-parser = { git = "ssh://git@github.com/eigerco/axelar-aleo.git", branch = "aleo-helper-libs" }
```

Comment:
@tilacog how to handle this? including a crate from private repo

Reply:
We can either try to publish just the `aleo-parser` to crates.io, if that's doable; or create a new public repo for it.  

I prefer the first option, because that crate is very self-contained and don't expose our contract logic, so it should be safe to do so.

We might need to check that with @asmie first.

Reply:
@tilacog the contract log is not encrypted when your contract is deployed on testnet, so there is nothing to protect when it comes to contract logic

Reply:
@asmie mentioned enabling access for this repo to eigergo/axelar-aleo using GitHub tokens/secrets. 

This seems like the cleanest approach to me, because wouldn’t have to deal with new repositories or crate publishing.